frontend/src/components/pages/ChoresPage.js

```diff
@@ -1,5 +1,8 @@
 import { React } from "react";
 import { Navigate, useNavigate } from "@tanstack/react-location";
+import { useState } from "react";
+import { useMemo } from "react";
```

Comment:
I think you can group lines 3 and 4 like:
`import { React, useState, useMemo } from "react"`

Reply:
ty I made the change